src/main/java/io/quarkus/registry/app/metrics/MetricsConfiguration.java

```diff
@@ -21,7 +23,9 @@ public MeterFilter enableHistogram() {
             public DistributionStatisticConfig configure(Meter.Id id, DistributionStatisticConfig config) {
                 if (id.getName().equals("http.server.requests")) {
                     return DistributionStatisticConfig.builder()
-                            .percentilesHistogram(true) // histogram buckets (e.g. prometheus histogram_quantile)
+                            .serviceLevelObjectives(
```

Comment:
There are 2 ways to do this.
1. using standard percentiles and the 276 standard prometheus buckets, where you can only define the min and max values to expect, thus constraining the amount of buckets:
```
            @Override
            public DistributionStatisticConfig configure(Meter.Id id, DistributionStatisticConfig config) {
                if (id.getName().equals("http.server.requests")) {
                    return DistributionStatisticConfig.builder()
                            .percentilesHistogram(true) // histogram buckets (e.g. prometheus histogram_quantile)
                            .minimumExpectedValue(TimeUnit.MILLISECONDS.toNanos(10)) // min value of the histogram buckets
                            .maximumExpectedValue(TimeUnit.SECONDS.toNanos(1)) // max value of the histogram buckets
                            .build()
                            .merge(config);
                }
                return config;
            }
```
This will generate ~32 buckets. 


2. explicitly defining some custom SLOs:
 ```
  @Override
            public DistributionStatisticConfig configure(Meter.Id id, DistributionStatisticConfig config) {
                if (id.getName().equals("http.server.requests")) {
                    return DistributionStatisticConfig.builder()
                            .percentilesHistogram(false) // histogram buckets (e.g. prometheus histogram_quantile)
                            .serviceLevelObjectives(
                                    TimeUnit.MILLISECONDS.toNanos(100),
                                    TimeUnit.MILLISECONDS.toNanos(200),
                                    TimeUnit.MILLISECONDS.toNanos(300),
                                    TimeUnit.MILLISECONDS.toNanos(400),
                                    TimeUnit.MILLISECONDS.toNanos(500),
                                    TimeUnit.MILLISECONDS.toNanos(600),
                                    TimeUnit.MILLISECONDS.toNanos(700),
                                    TimeUnit.MILLISECONDS.toNanos(800),
                                    TimeUnit.MILLISECONDS.toNanos(900),
                                    TimeUnit.MILLISECONDS.toNanos(1000)) //slo slots
                            .build()
                            .merge(config);
                }
                return config;
            }
 ```
This will generate 10 buckets, which is not very useful because the buckets are distributed linearly. 
---
If you can define a better lower band, I recommend the 1st option, even if it generates more buckets.

Reply:
Thank you very much! I really appreciate that! 
Although the 1st option seems better, I've noticed that other services in the AppSRE are using the second option, so I'll stick with it for now.